Lib/test/test_asyncio/test_tasks.py

```diff
@@ -80,6 +80,12 @@ def __await__(self):
         return self
 
 
+# Next value can be used as very small sleep time:
+# it passes check "sleep_time > 0", but has almost
```

Comment:
I think it would be better to say timeout or delay here instead of sleep time because it was the timeout inside `wait_for()` that has to be > 0, and it's being passed to `call_later()` rather than `sleep()`. (And often for 
 sleep, `sleep(0)` is sufficient so an epsilon isn't needed there.)